gdcdictionary/schemas/demographic.yaml

```diff
@@ -288,46 +214,61 @@ properties:
         term_id: C28554
         term_version: 18.11d (Release date:2018-11-26)
 
-  human_age_at_index:
+  age:
     description: >
-      The participant's age (in years) on the reference or anchor date used during date obfuscation. If this age is greater than 89 years, see 'human_age_at_index_gt89'.
-    type:
-      - number
-      - "null"
-    maximum: 89
-    minimum: 0
+      Please give age in either: 
+      -years (most common) 
+      -months 
+      -days
+    type: integer
+
+  age_unit:
+    description: >
+      Age units (yr/mo/days):
+    enum:
+      - Years
+      - Months
+      - Days
     termDef:
-      - term: Human Age At Index
-        source: NCI Thesaurus
-        term_id: C14225
-        term_version: 18.12e (Release date:2018-12-31)
-      - term: Human Age At Index
-        source: NCI Thesaurus
-        term_id: C37908
-        term_version: 18.12e (Release date:2018-12-31)
-      - term: Human Age At Index
+      - term: ageunit
         source: NCI Thesaurus
-        term_id: C25390
-        term_version: 18.12e (Release date:2018-12-31)
+        term_id: C50400
+        term_version: 20.03e (Release date:2020-03-31)
 
-  human_age_at_index_gt89:
+  res_county:
     description: >
-      Indicate if the participant's age (in years) on the reference or anchor date used during date obfuscation is greater than 89 years.
-    enum:
-      - "Yes"
-      - "No"
+      County of residence
+    type: string
 
-  nationality:
+  res_state:
```

Comment:
We decided that we would be using province_state to capture this information as indicated in the Core Props SS.